src/components/Layout/Header/PageHeader.tsx

```diff
@@ -1,35 +1,40 @@
 import { ArrowBackIcon } from '@chakra-ui/icons'
-import type { FlexProps, ResponsiveValue } from '@chakra-ui/react'
+import type { FlexProps, IconButtonProps, ResponsiveValue, TextProps } from '@chakra-ui/react'
 import { Flex, IconButton, SimpleGrid } from '@chakra-ui/react'
 import type { Property } from 'csstype'
 import type { PropsWithChildren } from 'react'
 import React from 'react'
 import { useHistory } from 'react-router'
+import { RawText } from 'components/Text'
 
 const arrowBack = <ArrowBackIcon />
 const paddingX = { base: 4, xl: 8 }
 const paddingTop = { base: 'env(safe-area-inset-top)', md: 6 }
 const position: ResponsiveValue<Property.Position> = { base: 'sticky', md: 'relative' }
 
-export const PageBackButton = () => {
+export const PageBackButton: React.FC<{ onBack?: () => void }> = ({ onBack }) => {
   const { goBack } = useHistory()
   return (
     <IconButton
       fontSize='2xl'
       variant='ghost'
-      order='1'
       aria-label='go back'
       isRound
-      onClick={goBack}
+      onClick={onBack ?? goBack}
```

Comment:
Do we always want one or the other, or do we always want to precede `onBack` with a `goBack` call?

Reply:
This is the weird stuff will mobile I was talking about where the tabs mess with the history. Sometimes we need to pass where it should go instead of using history.

Reply:
But yah you answer you question, if we don't pass an onBack then we use goBack. 

So we can assume if they use this component they either are going to use the history, or their own custom go back callback.